pkg/common/http.go

```diff
@@ -88,8 +89,15 @@ type CustomTransport struct {
 	T http.RoundTripper
 }
 
+func userAgent() string {
+	if len(feature.UserAgentSuffix.Load()) > 0 {
+		return "TruffleHog" + " " + feature.UserAgentSuffix.Load()
```

Comment:
Is there a reason for the explicit concatenation of the space `"TruffleHog" + " "`? Instead of a single string `"TruffleHog "`.

Reply:
Overlooked that, thank you